AEMG/training.py

```diff
@@ -43,11 +52,14 @@ def __init__(self, config, loaders, verbose):
 
         self.dynamics_train_loader = loaders['train_dynamics']
         self.dynamics_test_loader = loaders['test_dynamics']
-        self.labels_loader = loaders['labels']
+        self.labels_train_loader = loaders['train_labels']
+        self.labels_test_loader = loaders['test_labels']
 
         self.reset_losses()
 
-        self.criterion = nn.MSELoss(reduction='mean')
+        self.dynamics_criterion = nn.MSELoss(reduction='mean')
+        self.labels_criterion = LabelsLoss()
```

Comment:
`reduction='mean'`?

Reply:
There was a mistake in the loss function. I've modified line 34 to: 
`l2_norm = torch.mean(torch.linalg.vector_norm(x - y, ord=2, dim=1))`

So it will a mean reduction always

Reply:
Does using the mean of `1 - tanh(norm)` make sense? My intuition says the `reduction = 'mean'` would be better since the other loss is using that. Check [this](https://discuss.pytorch.org/t/loss-reduction-sum-vs-mean-when-to-use-each/115641/2) for e.g.